onboarding-checklist.md

```diff
@@ -0,0 +1,140 @@
+
+## Onboardee checklist ##
+
+### Getting to know VM Fusion ###
+
+- [ ] Take notes on anything that is confusing or frustrating during your
+onboarding process.
+  - Please share this information with your buddy and/or the team so we can
+  make things better.
+  - If you notice a problem (especially with things like documentation), you
+  are more than welcome to fix it!
+  - You can also file issues and pull requests on the [template onboarding checklist].
+- [ ] Meet with your onboarding buddy (they should reach out to you).
+- [ ] Once you've finished the checklists below, make suggestions for steps
+that would have improved your onboarding experience as pull requests on the
+[template onboarding checklist] used to make this issue.
+
+### Required items for all team members ###
+
+Completing these items helps us fulfill security and compliance requirements.
+If you get stuck, or if these requirements are confusing, ask for help from
+your onboarding buddy.
+
+Pre-requisites:
+
+- [ ] Complete [PALMS Cybersecurity Training](https://etms.hq.dhs.gov),
+including Mandatory Cyber Security and Privacy Training, including accepting
+the Rules of Behavior, which is required before we can give you access to any
+systems, like Trio.
+- [ ] Go through the [FISMA-Ready Github guide](https://github.com/fisma-ready/github)
+to set up your Github account to be added to CISA.
+
+### Microsoft Teams chat ###
+
+Once you're set up on Microsoft Teams, we recommend you join the following:
+
+- [ ] [CISA](https://teams.microsoft.com/l/team/19%3aa40fd338ea7a45de994a7c7ce9671fd4%40thread.skype/conversations?groupId=8c6c0b00-7244-46e1-bef9-76ce2a1906f5&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+- [ ] [CSD All Staff](https://teams.microsoft.com/l/team/19%3a65f1c4b58edf4246995ae3fa732767ee%40thread.skype/conversations?groupId=6dfc3f23-3a49-44ce-ad37-6c7a9d8fb0aa&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+- [ ] [Vulnerability Management](https://teams.microsoft.com/l/team/19%3a2161749c8eb94adeb31480e298f3b532%40thread.skype/conversations?groupId=61e939a6-2165-4085-a26e-decae97d1471&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+
+### Getting to know the VM Fusion Dev Team ###
+
+- [ ] Bookmark the [dev team project board] to track issues and work across
+multiple Github repositories.
+- [ ] Bookmark [VM JIRA] to work with the rest of VM Fusion.
+- [ ] In addition to your CISA email address `first.last@cisa.dhs.gov`, you'll
+also get an account on Trio, which is a Google-based account used for
+cross-team and cross-agency collaboration and is generally
+`first.last@trio.dhs.gov`. Trio may take a few days to get set up.
+  - Once you're set up on Trio, visit
+  [chat.google.com](https://chat.google.com) and join applicable rooms.
+- [ ] Make sure you have been added to several [dev team Google Groups] so you
+can participate in team-wide internal communication:
+  - [ ] [ALL Trio users@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/all)
```

Comment:
```suggestion
  - [ ] [ALL Trio users @ beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/all)
```
I thought it was an e-mail address at first glance.

Reply:
Perhaps "ALL Trio users at `beta.dhs.gov`" makes sense for these links?  Using the "@" seems superfluous here.

Reply:
I'd copied these from the google groups page, will simplify all of them for clarity.